source/sync/go-to-production/compact-disk.txt

```diff
@@ -4,62 +4,77 @@
 Compact an Atlas Volume
 =======================
 
-.. default-domain:: mongodb
+.. meta::
+  :description: Compact an Atlas volume to reclaim storage space for Atlas Device Sync data.
+
+.. facet::
+  :name: genre
+  :values: tutorial
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
+Atlas Device SDK's :ref:`data compaction <backend-compaction>` defragments the storage
+space the local database file used on a client and reduces the size of the
+``__realm_sync`` database on Atlas clusters. However, it does *not* reduce the
+overall size of the Atlas cluster. Compacting the data on an Atlas volume
+involves additional steps.
 
-Overview
---------
-
-While Realm :ref:`data compaction <backend-compaction>` defragments the storage 
-space the realm file uses on a *client*, and also reduces the size of the 
-``__realm_sync`` database on Atlas clusters, it does not reduce the overall size of 
-the Atlas cluster. Compacting the data on an Atlas volume involves several 
-additional steps. 
-
-Steps to Compact and Reclaim Disk Space
+Enable Compaction to Reclaim Disk Space
 ---------------------------------------
 
-You should use the following guidance for reducing Atlas disk space usage:  
+Perform the following to reduce Atlas disk space usage:
 
 1. Enable Compaction in App Services
 
-   Before reclaiming volume space, be sure that you have enabled Compaction 
-   on your Atlas App Services app. You set this by setting the Max Offline Time 
-   setting in the Atlas UI. For more information, see 
-   :ref:`<optimize-sync-atlas-usage>`.
+   Before reclaiming volume space, be sure that you have enabled Compaction
+   on your Atlas App Services app. To do so, set the :guilabel:`Client Max Offline Time`
```

Comment:
Isn't this on by default now? I would suggest we may want to mention you can skip this step unless you have explicitly disabled `Client Max Offline Time`.